src/NzbDrone.Core/MediaFiles/MediaFileRepository.cs

```diff
@@ -26,6 +27,11 @@ public List<EpisodeFile> GetFilesBySeries(int seriesId)
             return Query(c => c.SeriesId == seriesId).ToList();
         }
 
+        public List<EpisodeFile> GetFilesBySeries(List<int> seriesIds)
```

Comment:
This will better align with `EpisodeRepository`

```suggestion
        public List<EpisodeFile> GetFilesBySeriesIds(List<int> seriesIds)
```